app/views/schools/advice/advice_base/insights.html.erb

```diff
@@ -1,24 +1,28 @@
-<%= render 'schools/advice/advice_page', school: @school, advice_pages: @advice_pages, advice_page: @advice_page,
-                                         tab: @tab, data_warning: @data_warning do %>
+<%= cache_if @cache_view, [@school.id, @school.name.parameterize, @tab, @school.validation_cache_key].join('-') do %>
```

Comment:
sufficient to use `@school`? Think it'll use `@school.updated_at` then.  Believe template name is automatically used in the cache key so don't need @tab?

also wonder if we also want a long (day or week?) expires_in as otherwise not sure cache file would be cleaned up

Reply:
I'm going to revert this change here and introduce a separate PR with caching changes. Need to think a bit more about caching the entire partial or sections, e.g. the nav, _insights, _analysis, etc.

You're right about the keys. I think @school may be sufficient so long as we're updating the record to set the validation_cache_key. Hadn't realised about the template being included.